apps/meteor/client/lib/rooms/roomTypes/livechat.ts

```diff
@@ -70,6 +70,9 @@ roomCoordinator.add(LivechatRoomType, {
 	},
 
 	findRoom(identifier) {
+		if (!hasPermission('view-l-room')) {
+			return;
+		}
```

Comment:
```suggestion
```

It's better not to have this test at all. It'll only cause the room data to be loaded from the server every time.